Yoast/Yoast-SEO-for-TYPO3
# [TASK] Add last_mod field in page properties to set correct last_mod in Sitemap XML

## Summary

This PR can be summarized in the following changelog entry:

* Added a separate field to track the last change on the page. This date is not only set by changes in the page record, but also on update of content elements.

## Test instructions

This PR can be tested by following these steps:

* Check domain.com/sitemap.xml
* Do a change in page record
* Check domain.com/sitemap.xml if last_mod is changed
* Do a change in a content element on a page
* Check domain.com/sitemap.xml if last_mod of page where content element is on, is changed


## Quality assurance


Fixes #
